content-security-policy/blink-contrib-2/base-uri-allow.sub.html.sub.headers

```diff
@@ -0,0 +1,6 @@
+Expires: Mon, 26 Jul 1997 05:00:00 GMT
+Cache-Control: no-store, no-cache, must-revalidate
+Cache-Control: post-check=0, pre-check=0, false
+Pragma: no-cache
+Set-Cookie: base-uri-allow={{$id:uuid()}}; Path=/content-security-policy/blink-contrib-2
+Content-Security-Policy: base-uri http://w3c-test.org; script-src 'self' 'unsafe-inline' http://w3c-test.org; connect-src 'self'; report-uri /content-security-policy/support/report.py?op=put&reportID={{$id}}
```

Comment:
Will the base-uri present an issue when ran locally?
E.g. over http://web-platform.test:8000 ?